com.unity.render-pipelines.high-definition/Editor/AssetProcessors/MaterialPostProcessor.cs

```diff
@@ -402,7 +372,7 @@ static void MoreMaterialSurfaceOptionFromShaderGraph(Material material, HDShader
                 // Synchronize properties we exposed from SG to the material
                 ResetFloatProperty(kReceivesSSR);
                 ResetFloatProperty(kReceivesSSRTransparent);
-                ResetFloatProperty(kEnableDecals);
+                ResetFloatProperty("_SupportDecals");
```

Comment:
Isn't `_SupportDecals` the same as `kEnableDecals` ?

Reply:
because as I remove kEnableDecals  then I can use it :)